addons/api/mirage/generated/factories/host-set.js

```diff
@@ -5,7 +5,7 @@ import { random, date, datatype } from 'faker';
  * GeneratedHostSetModelFactory
  */
 export default Factory.extend({
-  type: 'static',
+  type: () => this.hostCatalog.type,
```

Comment:
Right logic.  However the inheritance is "undiscoverable" at the schema level, so this can't be generated.  I think the generated factories ought to express a "ground state" default type.  Inheritance can be expressed in the factory subclasses.

Reply:
Exactly!! I just fixed what a good timing, eh? I just ask you about this in another PR and after it, I realize I have that exact problem. SOLVED!